pypy/module/_vmprof/moduledef.py

```diff
@@ -20,6 +21,8 @@ class Module(MixedModule):
 
         'VMProfError': 'space.fromcache(interp_vmprof.Cache).w_VMProfError',
     }
+    if rvmprof.supports_native_profiling():
+        interpleveldefs.update({'resolve_addr': 'interp_vmprof.vmprof_resolve_address'})
```

Comment:
you could just write `interpleveldefs['resolve_addr'] = 'interp_vmprof.vmprof_resolve_address'` here